tools/shared.py

```diff
@@ -1167,6 +1165,9 @@ def load(self, args=[]):
           declare = re.sub(r'([\w\d]+)\s*=\s*(.+)', r'self.attrs["\1"]=\2;', args[i+1])
           exec declare
 
+      if get_llvm_target() == WASM_TARGET:
```

Comment:
Looking at the code one way to do that would be to remove WASM_BACKEND from settings.js completely..  and then anyone trying tor read it before this line would hit an assert because it would be missing.  But perhaps that version in settings.js (defaulting it to 0) is needed somwhere else?  (i.e. if setting.js is loaded standalone in node?)